<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Discussion | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)</title>
  <meta name="description" content="Chapter 3 Discussion | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)" />
  <meta name="generator" content="bookdown 0.26.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Discussion | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Discussion | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)" />
  
  
  

<meta name="author" content="Shruti Sridhar" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="2-results.html"/>
<link rel="next" href="conclusion.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#objective"><i class="fa fa-check"></i>Objective</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="1" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html"><i class="fa fa-check"></i><b>1</b> Methodology</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#datasets"><i class="fa fa-check"></i><b>1.1</b> Datasets</a></li>
<li class="chapter" data-level="1.2" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#immunedeconvolution"><i class="fa fa-check"></i><b>1.2</b> Immunedeconvolution</a></li>
<li class="chapter" data-level="1.3" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#pathway-analysis"><i class="fa fa-check"></i><b>1.3</b> Pathway Analysis</a></li>
<li class="chapter" data-level="1.4" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#sting-correlation-analysis"><i class="fa fa-check"></i><b>1.4</b> STING correlation analysis</a></li>
<li class="chapter" data-level="1.5" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#statistical-analysis"><i class="fa fa-check"></i><b>1.5</b> Statistical analysis</a></li>
<li class="chapter" data-level="1.6" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#clustering-analysis"><i class="fa fa-check"></i><b>1.6</b> Clustering analysis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-results.html"><a href="2-results.html"><i class="fa fa-check"></i><b>2</b> Results</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-results.html"><a href="2-results.html#pathway-analysis-1"><i class="fa fa-check"></i><b>2.1</b> Pathway Analysis</a></li>
<li class="chapter" data-level="2.2" data-path="2-results.html"><a href="2-results.html#sting-correlation-analysis-1"><i class="fa fa-check"></i><b>2.2</b> STING Correlation Analysis</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-results.html"><a href="2-results.html#violin-plot-of-tpm-distribution-of-sting-across-cibersort-quartiles-for-t-cell-subtypes"><i class="fa fa-check"></i><b>2.2.1</b> Violin plot of TPM distribution of STING across CIBERSORT quartiles for T cell subtypes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-results.html"><a href="2-results.html#violin-plot-of-tpm-distribution-of-sting-across-cibersort-quartiles-for-macrophage-subtypes"><i class="fa fa-check"></i><b>2.2.2</b> Violin plot of TPM distribution of STING across CIBERSORT quartiles for Macrophage subtypes</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-results.html"><a href="2-results.html#sting-gsea-analysis"><i class="fa fa-check"></i><b>2.3</b> STING GSEA Analysis</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-results.html"><a href="2-results.html#gsea-analysis-for-gse10846"><i class="fa fa-check"></i><b>2.3.1</b> GSEA analysis for GSE10846</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-results.html"><a href="2-results.html#gsea-analysis-for-gse23501"><i class="fa fa-check"></i><b>2.3.2</b> GSEA analysis for GSE23501</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-results.html"><a href="2-results.html#gsea-analysis-for-gse31312"><i class="fa fa-check"></i><b>2.3.3</b> GSEA analysis for GSE31312</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-results.html"><a href="2-results.html#gsea-analysis-for-gse32918"><i class="fa fa-check"></i><b>2.3.4</b> GSEA analysis for GSE32918</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-results.html"><a href="2-results.html#gsea-analysis-for-gse98588"><i class="fa fa-check"></i><b>2.3.5</b> GSEA analysis for GSE98588</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-results.html"><a href="2-results.html#gsea-analysis-for-gse117556"><i class="fa fa-check"></i><b>2.3.6</b> GSEA analysis for GSE117556</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-results.html"><a href="2-results.html#cell-line-analysis"><i class="fa fa-check"></i><b>2.4</b> Cell line analysis</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-results.html"><a href="2-results.html#farage-cell-line"><i class="fa fa-check"></i><b>2.4.1</b> Farage Cell Line</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-results.html"><a href="2-results.html#nu-dhl-1"><i class="fa fa-check"></i><b>2.4.2</b> NU-DHL-1</a></li>
<li class="chapter" data-level="2.4.3" data-path="2-results.html"><a href="2-results.html#oci-ly3"><i class="fa fa-check"></i><b>2.4.3</b> OCI-Ly3</a></li>
<li class="chapter" data-level="2.4.4" data-path="2-results.html"><a href="2-results.html#riva"><i class="fa fa-check"></i><b>2.4.4</b> RIVA</a></li>
<li class="chapter" data-level="2.4.5" data-path="2-results.html"><a href="2-results.html#u9322"><i class="fa fa-check"></i><b>2.4.5</b> U9322</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-results.html"><a href="2-results.html#gapdh-and-sting-correlation"><i class="fa fa-check"></i><b>2.5</b> GAPDH and STING Correlation</a></li>
<li class="chapter" data-level="2.6" data-path="2-results.html"><a href="2-results.html#tumor-clustering-ananlysis"><i class="fa fa-check"></i><b>2.6</b> Tumor Clustering Ananlysis</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="2-results.html"><a href="2-results.html#gse10846-clustering"><i class="fa fa-check"></i><b>2.6.1</b> GSE10846 Clustering</a></li>
<li class="chapter" data-level="2.6.2" data-path="2-results.html"><a href="2-results.html#gse117556-clustering"><i class="fa fa-check"></i><b>2.6.2</b> GSE117556 Clustering</a></li>
<li class="chapter" data-level="2.6.3" data-path="2-results.html"><a href="2-results.html#gse31312-clustering"><i class="fa fa-check"></i><b>2.6.3</b> GSE31312 Clustering</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-discussion.html"><a href="3-discussion.html"><i class="fa fa-check"></i><b>3</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-discussion.html"><a href="3-discussion.html#the-immune-microenvironment-of-dlbcl"><i class="fa fa-check"></i><b>3.1</b> The immune microenvironment of DLBCL</a></li>
<li class="chapter" data-level="3.2" data-path="3-discussion.html"><a href="3-discussion.html#sting-as-a-recruiter-of-immune-cells"><i class="fa fa-check"></i><b>3.2</b> STING as a recruiter of immune cells</a></li>
<li class="chapter" data-level="3.3" data-path="3-discussion.html"><a href="3-discussion.html#pathway-level-regulation-by-sting"><i class="fa fa-check"></i><b>3.3</b> Pathway level regulation by STING</a></li>
<li class="chapter" data-level="3.4" data-path="3-discussion.html"><a href="3-discussion.html#sting-and-its-role-in-the-lymophoma-microenvrionment"><i class="fa fa-check"></i><b>3.4</b> STING and its role in the lymophoma microenvrionment</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-appendix.html"><a href="A-appendix.html"><i class="fa fa-check"></i><b>A</b> Appendix</a>
<ul>
<li class="chapter" data-level="A.1" data-path="A-appendix.html"><a href="A-appendix.html#pathways-analysis"><i class="fa fa-check"></i><b>A.1</b> Pathways Analysis</a></li>
<li class="chapter" data-level="A.2" data-path="A-appendix.html"><a href="A-appendix.html#correlation-analysis"><i class="fa fa-check"></i><b>A.2</b> Correlation Analysis</a>
<ul>
<li class="chapter" data-level="A.2.1" data-path="A-appendix.html"><a href="A-appendix.html#dave"><i class="fa fa-check"></i><b>A.2.1</b> Dave</a></li>
<li class="chapter" data-level="A.2.2" data-path="A-appendix.html"><a href="A-appendix.html#staudt"><i class="fa fa-check"></i><b>A.2.2</b> Staudt</a></li>
<li class="chapter" data-level="A.2.3" data-path="A-appendix.html"><a href="A-appendix.html#goya"><i class="fa fa-check"></i><b>A.2.3</b> GOYA</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="discussion" class="section level1 hasAnchor" number="3">
<h1><span class="header-section-number">Chapter 3</span> Discussion<a href="3-discussion.html#discussion" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="the-immune-microenvironment-of-dlbcl" class="section level2 hasAnchor" number="3.1">
<h2><span class="header-section-number">3.1</span> The immune microenvironment of DLBCL<a href="3-discussion.html#the-immune-microenvironment-of-dlbcl" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The tumor microenvironment of DLBCL is known to contribute to tumor progression and immune evasion. The immune microenvironment includes several different cell types such as including NLCs/tumor-associated macrophages(TAMs), tumor-infiltrating lymphocytes(TILs), regulatory T cells, dendritic cells(DCs), myeloid suppressor cells(MSCs) and endothelial stromal cells, which interact with and are enlisted by malignant cells <span class="citation">(<a href="#ref-xu2021tumor" role="doc-biblioref">Xu et al., 2021</a>)</span>. The density of immune cells in the tumor and microenvironment is also known to be associated with prognosis.</p>
<p>Merdan et al <span class="citation">(<a href="#ref-merdan2021gene" role="doc-biblioref">Merdan et al., 2021</a>)</span> evaluated the CIBERSORT deconvolution scores for different immune sub types and stratified the scores according to clinical risk groups. Consistent with their results, the CIBERSORT analysis conducted showed that CD8+ T cells, CD4+ memory resting and macrophage populations were notably visible in the TME. Further they also established an association between the relative proportions of the immune cell types within subgroups of DLBCL patients by various clinical traits including cell of origin.</p>
<p>DLBCL arises from the germinal center and shows two distinct sub types: germinal-center B-cell-like (GCB) DLBCL and activated B-cell-like (ABC) DLBCL. Germinal-center B-cell-like (GCB) diffuse large B-cell lymphoma (DLBCL) are derived from normal germinal-center centroblasts whereas Activated B-cell-like (ABC) diffuse large B-cell lymphomas (DLBCLs) are derived from plasmablasts. <span class="citation">(<a href="#ref-monti2005molecular" role="doc-biblioref">Monti et al., 2005</a>)</span></p>
<p>Certain transformations of the GCB result in malignancy such as deregulation of BCL2, PI3K/mTOR pathway, p53 mutations and MDM2 gain/amplification <span class="citation">(<a href="#ref-frick2011molecular" role="doc-biblioref">Frick, Dörken, &amp; Lenz, 2011</a>)</span>. Deregulation of anti apoptotic BCL2 leads to overexpression of BCL2, making the cell resist cell death.</p>
<p>ABC DLBCL however, shows constitutive activation of the oncogenic nuclear factor kappa B (NF-kB) signaling pathway as well as de-regulation BCL2 protein <span class="citation">(<a href="#ref-frick2011molecular" role="doc-biblioref">Frick, Dörken, &amp; Lenz, 2011</a>)</span>. NF-κB activity is required for normal B cell development and survival. Lymphoid malignancies typically circumvent cell death by activation of the NFkB pathway. In particular for B cells, NFkB activation is seen downstream of several receptors including BCR, CD40, the B-cell–activating factor (BAFF) receptor, and Toll-like receptors (TLRs) <span class="citation">(<a href="#ref-lam2008cooperative" role="doc-biblioref">Lam et al., 2008</a>)</span></p>
<p>BCL6, a transcription repressor is also known to play a causative role in lymphomagenesis. STAT3 is a negative transcriptional regulator of BCL6. Sustained STAT3 activity (for proliferation and survival) is seen in ABC DLBCL cells, suggesting that chemoresistance and consequently poorer outcomes, exhibited by ABC DLBCL could be determined by simultaneous activation of both NFkB and STAT3 <span class="citation">(<a href="#ref-nowakowski2015abc" role="doc-biblioref">Nowakowski &amp; Czuczman, 2015</a>)</span></p>
<p>Further, lymphocytes and lymphoid malignancies depend on cytokine stimulation for growth and survival. B cell microenvironment interaction is influenced by cytokines such as interleukin (IL) 4, IL6, IL10 and IL21 <span class="citation">(<a href="#ref-beguelin2020mutant" role="doc-biblioref">Béguelin et al., 2020</a>)</span>.</p>
<p>It was seen that M0 macrophages, memory resting CD4 + T-cells and CD8+ T cells show greater infiltration particularly in GCB DLBCL samples. This could suggest that the pathways (such as complement, inflammatory response for M0, response to DNA damage for CD4+ memory resting and interferon gamma and alpha for CD8+ T cells) that show greater enrichment than STAT3 and NFkB (which are the pathways active in ABC DLBCL) could represent pathways active in GCB DLBCL that drive the infiltration of these cell types.</p>
</div>
<div id="sting-as-a-recruiter-of-immune-cells" class="section level2 hasAnchor" number="3.2">
<h2><span class="header-section-number">3.2</span> STING as a recruiter of immune cells<a href="3-discussion.html#sting-as-a-recruiter-of-immune-cells" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Overall, from the analysis it is seen that, T cell sub types - CD4+ memory activated, CD4+ memory resting, CD8+, follicular helper cells and Macrophage sub types - M0, M1 and M2 show consistent difference in STING expression and infiltration across all three datasets.</p>
<p>In different tumor models, activation of the cGAS-STING signal pathway was found to be significantly associated with infiltration of macrophages, CD4+ T cells, CD8+ T cells, B cells, and dendritic cells, as well as their immune markers <span class="citation">(<a href="#ref-qi2020identification" role="doc-biblioref">Qi et al., 2020</a>)</span></p>
<p>cGAMP activates the STING pathway, which is responsible for attracting macrophages into the tumor micro environment. M1 macrophages are known to have an anti tumor activity, while M2 are immunosuppressive. The STING pathway in macrophages is activated by NF-κβ and IRF3 followed by production of type I IFN within M1 cells, while another transcriptional factor, STAT6, is activated within M2 cells. STING has also been known to have a role in repolarizing tumor-promoting M2-type macrophages into M1-type macrophages. <span class="citation">(<a href="#ref-cheng2018nanoparticle" role="doc-biblioref">Cheng et al., 2018</a>)</span></p>
<p>Seng-Ryong Woo et al <span class="citation">(<a href="#ref-woo2014sting" role="doc-biblioref">Woo et al., 2014</a>)</span> reported that spontaneous T-cell priming was severely debilitated in STING-deficient and IRF3-deficient mice, which suggests that STING signaling may be necessary for the expansion of T cells. This implies that STING activation could facilitate T-cell priming and activation in the tumor micro environment. STING expression in endothelial cells is also known to be positively correlated with the infiltration level of CD8+ T cells and prolonged survival in several human cancer types (eg. colon and breast cancer) <span class="citation">(<a href="#ref-yang2019sting" role="doc-biblioref">H. Yang et al., 2019</a>)</span>. This suggests that STING activation in endothelial cells is necessary for the infiltration of CTLs.</p>
<p>Taken together, STING promotes the infiltration of various immune cells and also has an impact on macrophage polarization suggesting that further study is required to explore if STING can switch the TME from “cold” to “hot”</p>
</div>
<div id="pathway-level-regulation-by-sting" class="section level2 hasAnchor" number="3.3">
<h2><span class="header-section-number">3.3</span> Pathway level regulation by STING<a href="3-discussion.html#pathway-level-regulation-by-sting" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Hayman et al had previously highlighted a role for STING as a modulator for ROS homeostasis that influenced therapeutic sensitivity <span class="citation">(<a href="#ref-hayman2021sting" role="doc-biblioref">Hayman et al., 2021</a>)</span>. This was an interesting context to explore to understand if STING high tumors showed enriched genes that influence ROS production.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:STINGROS"></span>
<img src="figure/STING_ROS.png" alt="Proposed model of tumor intrinsic STING loss leading to treatment resistant phenotype" width="30%" height="30%" />
<p class="caption">
Figure 3.1: Proposed model of tumor intrinsic STING loss leading to treatment resistant phenotype
</p>
</div>
<p>Oxidative phosphorylation was found to be enriched in low STING samples for datasets GSE10846, GSE117556, GSE32918, GSE23501 (FDR &lt; 0.1). There is a very intricate relationship between OxPhos and ROS pathways and future analysis would be to try and understand the interplay between these two pathways and its correlation with prognosis and to establish consistency with Heyman et al.</p>
<p>Additionally, the pathways in common across all datasets included ‘Inflammatory Response,’ ‘IL6 -JAK-STAT3 Signalling’ and ‘Apical Junction’ pathways. The enrichment of inflammatory pathways in high STING samples is consistent with activation of STING promoting chemokine production that recruits several inflammatory factors including M2 tumor associated macrophages <span class="citation">(<a href="#ref-pepin2017cgas" role="doc-biblioref">Pépin &amp; Gantier, 2017</a>)</span>. Further analysis is required to understand the nature of regulation of these pathways in STING high tumors and possible mechanisms of action and its correlation, if any,with prognosis.</p>
<p>The cell line analysis showed a considerable overlap between pathways that were enriched in high STING samples and pathways enriched after chemotherapy treatment. Several studies suggested that activation of the cGAS-STING pathway has additional benefits in promoting immunostimulatory effects of chemotherapy drugs while reducing toxicity. A study by Al-Asmari et al <span class="citation">(<a href="#ref-al2022pharmacological" role="doc-biblioref">Al-Asmari et al., 2022</a>)</span> showed that both NF-κB and IRF3 branches are frequently engaged in cancer cells upon genotoxic treatment and that IL-6 induced by DNA damage was partially dependent on STING signaling. This also aligns with the literature supporting that IL-6 and its activation of STAT3 counteracts the effects of radio- and chemotherapy in many cancers <span class="citation">(<a href="#ref-yang2020stat3" role="doc-biblioref">P.-L. Yang, Liu, Li, &amp; Xu, 2020</a>)</span> in line with our hypothesis that the STING pathway increases signalling during chemotherapy.</p>
</div>
<div id="sting-and-its-role-in-the-lymophoma-microenvrionment" class="section level2 hasAnchor" number="3.4">
<h2><span class="header-section-number">3.4</span> STING and its role in the lymophoma microenvrionment<a href="3-discussion.html#sting-and-its-role-in-the-lymophoma-microenvrionment" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The datasets analyzed show a few consistent patterns. Firstly, a common trend of three major clusters is obtained. These are similar to the Depleted, Mesenchymal and Inflammatory clusters previously described by Kotlov et al. Second, STING high samples cluster together showing enriched inflammatory and stromal pathways and STING low samples cluster together showing overall depleted microenvironmental signatures.</p>
<p>As mentioned previously the enrichment of inflammatory pathways in the LME for STING high tumors can be attributed to the recruitment of inflammatory factors. Another member of the tumor microenvironment is stromal cells (endothelial cells and fibroblasts) that also express STING genes. Tumor-derived DNA can be transferred from cell to cell by phagocytosis of apoptotic bodies, activating the cGAS-STING signaling pathway in the cytoplasm, inducing type I IFN production.<span class="citation">(<a href="#ref-pepin2017cgas" role="doc-biblioref">Pépin &amp; Gantier, 2017</a>)</span></p>
<p>Activation of the STING signaling pathway in stromal cells induces vascular remodeling. Intratumoral STING activation normalizes the tumor vasculature and the tumor microenvironment. After STING activation in tumors, interferon-β (IFN-β) production plays a critical role in angiogenesis. IFN-β, an anti-angiogenic cytokine, inhibits endothelial cell proliferation and capillary network formation, up-regulates vascular normalized gene expression, normalizes the tumor vascular system, promotes CD8+ T cell infiltration, and ultimately enhances antitumor immunity <span class="citation">(<a href="#ref-pu2021immune" role="doc-biblioref">Pu, Chen, Liu, Zhang, &amp; Shao, 2021</a>)</span>.This is reflected in our analysis by samples with high STING expression showing enriched ECM and stromal pathways.</p>
<p>Further the survival analysis conducted by Kotlov et al <span class="citation">(<a href="#ref-kotlov2021clinical" role="doc-biblioref">Kotlov et al., 2021</a>)</span> for the various LME signatures shows that LME-Depleted had the highest proportion of high (4–5) international prognostic index (IPI) scores and analysis of OS by LME indicated significant differences in prognosis from better to poor as follows: MS (P=0.03, vs. GC-like), IN (P=0.0008, vs. MS), and DP (P=0.007, vs. IN) LMEs. This could hint at the role of STING (particularly higher expression of STING) in contributing to a favorable prognosis. This might also suggest a loss/lower STING expression could contribute to an initially immune-rich DLBCL showing an immune-deserted microenvironment associated with disease progression and poorer prognosis.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent" line-spacing="2">
<div id="ref-al2022pharmacological" class="csl-entry">
Al-Asmari, S. S., Rajapakse, A., Ullah, T. R., Pépin, G., Croft, L. V., &amp; Gantier, M. P. (2022). Pharmacological targeting of STING-dependent IL-6 production in cancer cells. <em>Frontiers in Cell and Developmental Biology</em>, 3835.
</div>
<div id="ref-beguelin2020mutant" class="csl-entry">
Béguelin, W., Teater, M., Meydan, C., Hoehn, K. B., Phillip, J. M., Soshnev, A. A., … others. (2020). Mutant EZH2 induces a pre-malignant lymphoma niche by reprogramming the immune response. <em>Cancer Cell</em>, <em>37</em>(5), 655–673.
</div>
<div id="ref-cheng2018nanoparticle" class="csl-entry">
Cheng, N., Watkins-Schulz, R., Junkins, R. D., David, C. N., Johnson, B. M., Montgomery, S. A., … others. (2018). A nanoparticle-incorporated STING activator enhances antitumor immunity in PD-L1–insensitive models of triple-negative breast cancer. <em>JCI Insight</em>, <em>3</em>(22).
</div>
<div id="ref-frick2011molecular" class="csl-entry">
Frick, M., Dörken, B., &amp; Lenz, G. (2011). The molecular biology of diffuse large b-cell lymphoma. <em>Therapeutic Advances in Hematology</em>, <em>2</em>(6), 369–379.
</div>
<div id="ref-hayman2021sting" class="csl-entry">
Hayman, T. J., Baro, M., MacNeil, T., Phoomak, C., Aung, T. N., Cui, W., … others. (2021). STING enhances cell death through regulation of reactive oxygen species and DNA damage. <em>Nature Communications</em>, <em>12</em>(1), 1–14.
</div>
<div id="ref-kotlov2021clinical" class="csl-entry">
Kotlov, N., Bagaev, A., Revuelta, M. V., Phillip, J. M., Cacciapuoti, M. T., Antysheva, Z., … others. (2021). Clinical and biological subtypes of b-cell lymphoma revealed by microenvironmental signatures. <em>Cancer Discovery</em>, <em>11</em>(6), 1468–1489.
</div>
<div id="ref-lam2008cooperative" class="csl-entry">
Lam, L. T., Wright, G., Davis, R. E., Lenz, G., Farinha, P., Dang, L., … Staudt, L. M. (2008). Cooperative signaling through the signal transducer and activator of transcription 3 and nuclear factor-<span class="math inline">\(\kappa\)</span>b pathways in subtypes of diffuse large b-cell lymphoma. <em>Blood, The Journal of the American Society of Hematology</em>, <em>111</em>(7), 3701–3713.
</div>
<div id="ref-merdan2021gene" class="csl-entry">
Merdan, S., Subramanian, K., Ayer, T., Van Weyenbergh, J., Chang, A., Koff, J. L., &amp; Flowers, C. (2021). Gene expression profiling-based risk prediction and profiles of immune infiltration in diffuse large b-cell lymphoma. <em>Blood Cancer Journal</em>, <em>11</em>(1), 1–12.
</div>
<div id="ref-monti2005molecular" class="csl-entry">
Monti, S., Savage, K. J., Kutok, J. L., Feuerhake, F., Kurtin, P., Mihm, M., … others. (2005). Molecular profiling of diffuse large b-cell lymphoma identifies robust subtypes including one characterized by host inflammatory response. <em>Blood</em>, <em>105</em>(5), 1851–1861.
</div>
<div id="ref-nowakowski2015abc" class="csl-entry">
Nowakowski, G. S., &amp; Czuczman, M. S. (2015). ABC, GCB, and double-hit diffuse large b-cell lymphoma: Does subtype make a difference in therapy selection? <em>American Society of Clinical Oncology Educational Book</em>, <em>35</em>(1), e449–e457.
</div>
<div id="ref-pepin2017cgas" class="csl-entry">
Pépin, G., &amp; Gantier, M. P. (2017). cGAS-STING activation in the tumor microenvironment and its role in cancer immunity. <em>Regulation of Inflammatory Signaling in Health and Disease</em>, 175–194.
</div>
<div id="ref-pu2021immune" class="csl-entry">
Pu, F., Chen, F., Liu, J., Zhang, Z., &amp; Shao, Z. (2021). Immune regulation of the cGAS-STING signaling pathway in the tumor microenvironment and its clinical application. <em>OncoTargets and Therapy</em>, <em>14</em>, 1501.
</div>
<div id="ref-qi2020identification" class="csl-entry">
Qi, Z., Yan, F., Chen, D., Xing, W., Li, Q., Zeng, W., … Xie, J. (2020). Identification of prognostic biomarkers and correlations with immune infiltrates among cGAS-STING in hepatocellular carcinoma. <em>Bioscience Reports</em>, <em>40</em>(10).
</div>
<div id="ref-woo2014sting" class="csl-entry">
Woo, S.-R., Fuertes, M. B., Corrales, L., Spranger, S., Furdyna, M. J., Leung, M. Y., … others. (2014). STING-dependent cytosolic DNA sensing mediates innate immune recognition of immunogenic tumors. <em>Immunity</em>, <em>41</em>(5), 830–842.
</div>
<div id="ref-xu2021tumor" class="csl-entry">
Xu, T., Chai, J., Wang, K., Jia, Q., Liu, Y., Wang, Y., … others. (2021). Tumor immune microenvironment components and checkpoint molecules in anaplastic variant of diffuse large b-cell lymphoma. <em>Frontiers in Oncology</em>, <em>11</em>.
</div>
<div id="ref-yang2019sting" class="csl-entry">
Yang, H., Lee, W. S., Kong, S. J., Kim, C. G., Kim, J. H., Chang, S. K., … others. (2019). STING activation reprograms tumor vasculatures and synergizes with VEGFR2 blockade. <em>The Journal of Clinical Investigation</em>, <em>129</em>(10), 4350–4364.
</div>
<div id="ref-yang2020stat3" class="csl-entry">
Yang, P.-L., Liu, L.-X., Li, E.-M., &amp; Xu, L.-Y. (2020). Stat3, the challenge for chemotherapeutic and radiotherapeutic efficacy. <em>Cancers</em>, <em>12</em>(9), 2459.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="2-results.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="conclusion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
