<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Methodology | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)</title>
  <meta name="description" content="Methodology | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)" />
  <meta name="generator" content="bookdown 0.26.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Methodology | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Methodology | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)" />
  
  
  

<meta name="author" content="Shruti Sridhar" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="introduction.html"/>
<link rel="next" href="results-and-discussion.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#objective"><i class="fa fa-check"></i>Objective</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="methodology.html"><a href="methodology.html"><i class="fa fa-check"></i>Methodology</a>
<ul>
<li class="chapter" data-level="0.1" data-path="methodology.html"><a href="methodology.html#datasets"><i class="fa fa-check"></i><b>0.1</b> Datasets</a></li>
<li class="chapter" data-level="0.2" data-path="methodology.html"><a href="methodology.html#immunedeconvolution"><i class="fa fa-check"></i><b>0.2</b> Immunedeconvolution</a></li>
<li class="chapter" data-level="0.3" data-path="methodology.html"><a href="methodology.html#pathway-analysis"><i class="fa fa-check"></i><b>0.3</b> Pathway Analysis</a></li>
<li class="chapter" data-level="0.4" data-path="methodology.html"><a href="methodology.html#sting-correlation-analysis"><i class="fa fa-check"></i><b>0.4</b> STING correlation analysis</a></li>
<li class="chapter" data-level="0.5" data-path="methodology.html"><a href="methodology.html#statistical-analysis"><i class="fa fa-check"></i><b>0.5</b> Statistical analysis</a></li>
<li class="chapter" data-level="0.6" data-path="methodology.html"><a href="methodology.html#clustering-analysis"><i class="fa fa-check"></i><b>0.6</b> Clustering analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="results-and-discussion.html"><a href="results-and-discussion.html"><i class="fa fa-check"></i>Results and Discussion</a>
<ul>
<li class="chapter" data-level="0.7" data-path="results-and-discussion.html"><a href="results-and-discussion.html#pathway-analysis-1"><i class="fa fa-check"></i><b>0.7</b> Pathway Analysis</a></li>
<li class="chapter" data-level="0.8" data-path="results-and-discussion.html"><a href="results-and-discussion.html#sting-cgas-pathway"><i class="fa fa-check"></i><b>0.8</b> STING cGAS Pathway</a></li>
<li class="chapter" data-level="0.9" data-path="results-and-discussion.html"><a href="results-and-discussion.html#sting-correlation-analysis-1"><i class="fa fa-check"></i><b>0.9</b> STING Correlation Analysis</a></li>
<li class="chapter" data-level="0.10" data-path="results-and-discussion.html"><a href="results-and-discussion.html#sting-gsea-analysis"><i class="fa fa-check"></i><b>0.10</b> STING GSEA Analysis</a>
<ul>
<li class="chapter" data-level="0.10.1" data-path="results-and-discussion.html"><a href="results-and-discussion.html#gsea-analysis-for-gse10846"><i class="fa fa-check"></i><b>0.10.1</b> GSEA analysis for GSE10846</a></li>
<li class="chapter" data-level="0.10.2" data-path="results-and-discussion.html"><a href="results-and-discussion.html#gsea-analysis-for-gse23501"><i class="fa fa-check"></i><b>0.10.2</b> GSEA analysis for GSE23501</a></li>
<li class="chapter" data-level="0.10.3" data-path="results-and-discussion.html"><a href="results-and-discussion.html#gsea-analysis-for-gse31312"><i class="fa fa-check"></i><b>0.10.3</b> GSEA analysis for GSE31312</a></li>
<li class="chapter" data-level="0.10.4" data-path="results-and-discussion.html"><a href="results-and-discussion.html#gsea-analysis-for-gse32918"><i class="fa fa-check"></i><b>0.10.4</b> GSEA analysis for GSE32918</a></li>
<li class="chapter" data-level="0.10.5" data-path="results-and-discussion.html"><a href="results-and-discussion.html#gsea-analysis-for-gse98588"><i class="fa fa-check"></i><b>0.10.5</b> GSEA analysis for GSE98588</a></li>
<li class="chapter" data-level="0.10.6" data-path="results-and-discussion.html"><a href="results-and-discussion.html#gsea-analysis-for-gse117556"><i class="fa fa-check"></i><b>0.10.6</b> GSEA analysis for GSE117556</a></li>
</ul></li>
<li class="chapter" data-level="0.11" data-path="results-and-discussion.html"><a href="results-and-discussion.html#gapdh-and-sting-correlation"><i class="fa fa-check"></i><b>0.11</b> GAPDH and STING Correlation</a></li>
<li class="chapter" data-level="0.12" data-path="results-and-discussion.html"><a href="results-and-discussion.html#tumor-clustering-ananlysis"><i class="fa fa-check"></i><b>0.12</b> Tumor Clustering Ananlysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-appendix.html"><a href="A-appendix.html"><i class="fa fa-check"></i><b>A</b> Appendix</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="methodology" class="section level1 unnumbered hasAnchor">
<h1>Methodology<a href="methodology.html#methodology" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="datasets" class="section level2 hasAnchor" number="0.1">
<h2><span class="header-section-number">0.1</span> Datasets<a href="methodology.html#datasets" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The following whole genome RNA-Seq DLBCL datasets were compiled for analysis and validation -</p>
<ul>
<li><strong>GOYA DLBCL dataset</strong> <span class="citation">(<a href="#ref-nowicka2021prognostic" role="doc-biblioref">Nowicka et al., 2021</a>)</span></li>
</ul>
<p>This dataset contains collected RNASeq data of 552 samples from the GOYA clinical trial. The GOYA trial tested the efficacy of Gazyva (GA101) compared with Rituxan (Rituximab) in first line, untreated DLBCL patients.</p>
<ul>
<li><strong>Dave dataset</strong> <span class="citation">(<a href="#ref-reddy2017genetic" role="doc-biblioref">Reddy et al., 2017</a>)</span></li>
</ul>
<p>This dataset contain exome sequencing data for 1001 DLBCL patients and RNA sequencing data for 775 DLBCL patients.</p>
<ul>
<li><strong>Staudt dataset</strong> <span class="citation">(<a href="#ref-schmitz2018genetics" role="doc-biblioref">Schmitz et al., 2018</a>)</span></li>
</ul>
<p>This retrospective study included 181 clinical samples from CHOP-treated patients and 233 clinical samples from Rituximab-CHOP-treated patients.</p>
<p>The following microarray datasets were choosen for analysis -</p>
</div>
<div id="immunedeconvolution" class="section level2 hasAnchor" number="0.2">
<h2><span class="header-section-number">0.2</span> Immunedeconvolution<a href="methodology.html#immunedeconvolution" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Immune deconvolution by the following methods was conducted for the chosen datasets.</p>
<ul>
<li><strong>quanTIseq</strong> <span class="citation">(<a href="#ref-finotello2019molecular" role="doc-biblioref">Finotello et al., 2019</a>)</span></li>
</ul>
<p>Computational pipeline for the quantification of the Tumor Immune contexture from human RNA-seq data</p>
<ul>
<li><strong>CIBERSORT</strong> <span class="citation">(<a href="#ref-chen2018profiling" role="doc-biblioref">Chen, Khodadoust, Liu, Newman, &amp; Alizadeh, 2018</a>)</span></li>
</ul>
<p>CIBERSORT takes an input matrix of reference gene expression signatures which is collectively used to estimate the relative proportions of each cell type of interest. CIBERSORT exhibits substantially improved accuracy for the analysis of mixture</p>
<ul>
<li><strong>MCP</strong> - Counter <span class="citation">(<a href="#ref-becht2016estimating" role="doc-biblioref">Becht et al., 2016</a>)</span></li>
</ul>
<p>Microenvironment Cell Populations-counter (MCP-counter) method, allows the quantification of the absolute abundance of eight immune and two stromal cell populations in heterogeneous tissues from transcriptomic data.</p>
<ul>
<li><strong>xCell</strong> <span class="citation">(<a href="#ref-widodo2021toward" role="doc-biblioref">Widodo et al., 2021</a>)</span></li>
</ul>
<p>xCell, a gene signature-based method that can be used to infer 64 immune and stromal cell types.</p>
<ul>
<li><strong>EPIC</strong> <span class="citation">(<a href="#ref-racle2020epic" role="doc-biblioref">Racle &amp; Gfeller, 2020</a>)</span></li>
</ul>
<p>Estimates the Proportion of Immune and Cancer cells (EPIC) from bulk tumor gene expression data. EPIC is based on a unique collection of RNA-Seq reference gene expression profiles from either circulating immune cells or tumor- infiltrating non-malignant cell types (i.e., immune, stromal and endothelial cells).</p>
</div>
<div id="pathway-analysis" class="section level2 hasAnchor" number="0.3">
<h2><span class="header-section-number">0.3</span> Pathway Analysis<a href="methodology.html#pathway-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>CIBERSORT results were chosen for further downstream analysis due to better resolution of the immune subtypes. For each immune subtype, the patient samples were divided into two cohorts (high and low) based on the immune cell subtype infiltration score (CIBERSORT Score). DeSeq223 analysis was conducted to find the differentially expressed genes following which Gene Set Enrichment24 Analysis using Hallmark gene sets was performed. R 4.1.1 and GraphPad PRISM 9 were used for visualization.</p>
</div>
<div id="sting-correlation-analysis" class="section level2 hasAnchor" number="0.4">
<h2><span class="header-section-number">0.4</span> STING correlation analysis<a href="methodology.html#sting-correlation-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The CIBERSORT Score for each immune cell type was divided into quartiles (CIBERSORT high (Q4), Interquartile range (IQR) and CIBERSORT low (Q1)) and correlated with STING TPM counts for each dataset. Violin plots were used to visualize the results. STING expression for each dataset was divided into quartiles and GSEA analysis was performed between Quartile 1 and Quartile 4 in each of the microarray datasets in order to evaluate pathway level differences.</p>
</div>
<div id="statistical-analysis" class="section level2 hasAnchor" number="0.5">
<h2><span class="header-section-number">0.5</span> Statistical analysis<a href="methodology.html#statistical-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Descriptive statistics and one way ANOVA tests were conducted using GraphPad PRISM 9 software for evaluating statistical significance of each quartile. FDR &lt; 0.05 and p value &lt; 0.05 were considered statistically significant for GSEA and GO analysis. Pearson correlation was conducted using R 4.1.1.</p>
</div>
<div id="clustering-analysis" class="section level2 hasAnchor" number="0.6">
<h2><span class="header-section-number">0.6</span> Clustering analysis<a href="methodology.html#clustering-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>25 FGES (Functional Gene Enrichment Signatures) signatures38 were used to identify microenvironmental patterns among DLBCL samples, stratified by STING expression (quartile 4 being STING High samples and quartile 1 Low STING samples) by unsupervised dense clustering using the Louvain method for community detection. Datasets GSE117556, GSE32918, GSE10846, GSE23501 and GSE31312 were individually analyzed and Gene Set Variation Analysis (GSVA) was conducted to generate activity scores.</p>
<p>Intersample similarity of the activity scores was calculated using Pearson correlation. The resulting distance matrix was converted into a graph where each sample formed a node and two nodes formed an edge with weight equal to the pair’s Pearson correlation.</p>
<p>To mathematically determine the optimum threshold for observed clusters for each dataset, maximum Calinski Harabasz and Silhouette scores were used excluding separations with low-populated clusters. The Calinski-Harabasz index, also known as the Variance Ratio Criterion, is the ratio of the sum of between-clusters dispersion and of inter-cluster dispersion for all clusters, the higher the score, the better the performance.</p>
<p>This score is fast to compute and the score is higher when clusters are dense and well separated. Silhouette Coefficient or silhouette score is a metric used to calculate the goodness of a clustering technique. Its value ranges from -1 to 1. WIth 1 meaning the clusters are well apart from each other and clearly distinguished. 0 meaning the clusters are indifferent and -1 meaning the clusters are assigned in the wrong way. The Silhouette score was used as validation after computing the Calinski-Harabasz index and optimum threshold was decided.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent" line-spacing="2">
<div id="ref-becht2016estimating" class="csl-entry">
Becht, E., Giraldo, N. A., Lacroix, L., Buttard, B., Elarouci, N., Petitprez, F., … others. (2016). Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression. <em>Genome Biology</em>, <em>17</em>(1), 1–20.
</div>
<div id="ref-chen2018profiling" class="csl-entry">
Chen, B., Khodadoust, M. S., Liu, C. L., Newman, A. M., &amp; Alizadeh, A. A. (2018). Profiling tumor infiltrating immune cells with CIBERSORT. In <em>Cancer systems biology</em> (pp. 243–259). Springer.
</div>
<div id="ref-finotello2019molecular" class="csl-entry">
Finotello, F., Mayer, C., Plattner, C., Laschober, G., Rieder, D., Hackl, H., … others. (2019). Molecular and pharmacological modulators of the tumor immune contexture revealed by deconvolution of RNA-seq data. <em>Genome Medicine</em>, <em>11</em>(1), 1–20.
</div>
<div id="ref-nowicka2021prognostic" class="csl-entry">
Nowicka, M., Hilton, L. K., Ashton-Key, M., Hargreaves, C. E., Lee, C., Foxall, R., … others. (2021). Prognostic significance of FCGR2B expression for the response of DLBCL patients to rituximab or obinutuzumab treatment. <em>Blood Advances</em>, <em>5</em>(15), 2945–2957.
</div>
<div id="ref-racle2020epic" class="csl-entry">
Racle, J., &amp; Gfeller, D. (2020). EPIC: A tool to estimate the proportions of different cell types from bulk gene expression data. In <em>Bioinformatics for cancer immunotherapy</em> (pp. 233–248). Springer.
</div>
<div id="ref-reddy2017genetic" class="csl-entry">
Reddy, A., Zhang, J., Davis, N. S., Moffitt, A. B., Love, C. L., Waldrop, A., … others. (2017). Genetic and functional drivers of diffuse large b cell lymphoma. <em>Cell</em>, <em>171</em>(2), 481–494.
</div>
<div id="ref-schmitz2018genetics" class="csl-entry">
Schmitz, R., Wright, G. W., Huang, D. W., Johnson, C. A., Phelan, J. D., Wang, J. Q., … others. (2018). Genetics and pathogenesis of diffuse large b-cell lymphoma. <em>New England Journal of Medicine</em>, <em>378</em>(15), 1396–1407.
</div>
<div id="ref-widodo2021toward" class="csl-entry">
Widodo, S. S., Hutchinson, R. A., Fang, Y., Mangiola, S., Neeson, P. J., Darcy, P. K., … others. (2021). Toward precision immunotherapy using multiplex immunohistochemistry and in silico methods to define the tumor immune microenvironment. <em>Cancer Immunology, Immunotherapy</em>, <em>70</em>(7), 1811–1820.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="results-and-discussion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
